app/build.gradle.kts

```diff
@@ -148,3 +151,15 @@ dependencies {
 	testImplementation(libs.koin.test)
 	testImplementation(libs.koin.test.junit4)
 }
+
+tasks.withType<Test> {
+    useJUnit()
+
+    testLogging {
+        events(FAILED, STANDARD_ERROR, SKIPPED)
+        exceptionFormat = FULL
```

Comment:
We don't use imports in our Gradle files, use the complete package instead like so:

```
exceptionFormat = org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
```